modules/microshift-updates-edge-to-image-uid-drift.adoc

```diff
@@ -0,0 +1,38 @@
+//Module included in the following assemblies:
+//
+//*  microshift_updating/microshift-update-rhel-edge-to-image-mode.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-updates-edge-to-image-uid-drift_{context}"]
+= Working around UID and GID drift when migrating to {op-system-image}
+
+Update procedures specific to {microshift-short} include planning for edge devices. If you do not re-install devices from scratch, you must use a workaround for UID and GID drift on edge devices during the migration process. One such workaround is to add systemd units that apply the necessary fixes by running before the affected system services.
```

Comment:
I do not understand what we're trying to say with: Update procedures specific to {microshift-short} include planning for edge devices.
Can we rephrase?

Reply:
```suggestion
If you do not re-install devices from scratch, you must use a workaround for a possible UID and GID drift on edge devices during the migration process. One way to solve this problem is to add systemd units that apply the necessary fixes before the affected system services are started.
```

Reply:
This is all around that same question. Updated, see what you think.